dfttk/vasp_input.py

```diff
@@ -243,30 +214,3 @@ def ev_curve_set(
 
     incar = Incar(incar_settings)
     incar.write_file(os.path.join(path, "INCAR"))
-
-
-def perturb_structure(
```

Comment:
We don't need this anymore? If it's the best way we have to do this sort of thing, we might keep it until later.

Reply:
I can remove it.